tests/app/main/test_users.py

```diff
@@ -0,0 +1,50 @@
+import mock
+from nose.tools import assert_equal, assert_in, assert_not_in
+from tests.app.helpers import BaseApplicationTest
+
+
+class TestListUsers(BaseApplicationTest):
+
+    @mock.patch('app.main.views.users.data_api_client')
+    def test_shows_services_list(self, data_api_client):
+        with self.app.test_client():
+            self.login()
+
+            data_api_client.find_users.return_value = {
+                'users': [{
+                    'id': 1,
+                    'name': "Don Draper",
+                    'emailAddress': "don@scdp.com",
+                    'loggedInAt': "2015-05-06T11:57:28.008690Z",
+                    'locked': False,
+                    'supplier': {
+                        'name': "Supplier Name",
+                        'supplierId': 1234
+                    }
+                }]
+            }
+
+            res = self.client.get('/suppliers/users')
+            assert_equal(res.status_code, 200)
+            data_api_client.find_users.assert_called_once_with(
```

Comment:
This doesn't need to change I'm just pointing it out because I remembered it yesterday. We now have a line length limit of 120 characters so this could be one line.